frappe/core/doctype/doctype/doctype.py

```diff
@@ -483,35 +483,35 @@ def check_illegal_characters(fieldname):
 	def check_unique_fieldname(fieldname):
 		duplicates = list(filter(None, map(lambda df: df.fieldname==fieldname and str(df.idx) or None, fields)))
 		if len(duplicates) > 1:
-			frappe.throw(_("Fieldname {0} appears multiple times in rows {1}").format(fieldname, ", ".join(duplicates)))
+			frappe.throw(_("[{0}]: Fieldname {1} appears multiple times in rows {2}").format(self.name, fieldname, ", ".join(duplicates)))
```

Comment:
I think we can remove the Square brackets for the Doctype

Reply:
done